operations/helm/charts/alloy/values.yaml

```
@@ -269,7 +271,9 @@ controller:
 
   # -- Additional containers to run alongside the Alloy container and initContainers.
   extraContainers: []
-
+  # -- terminationGracePeriodSeconds 
+  terminationGracePeriodSeconds: {}
```

```suggestion
  terminationGracePeriodSeconds: 
```
This is an integer, but by setting the default to `{}` it seems like it's supposed to be an object.